bool started
bool cancelled
bool config_saved
